hubot-scripts.json

```diff
@@ -19,5 +19,6 @@
   "rands-when-you-say.coffee",
   "scotch.coffee",
   "uptime.coffee",
-  "wikipedia.coffee"
+  "wikipedia.coffee",
+  "awwyiss.coffee"
```

Comment:
You actually don't need this line - anything in scripts/ gets added by default.

Can you remove that, rebase and I'll merge?